src/util/password.js

```diff
@@ -1,15 +1,16 @@
 var logger = require("../log/logger");
 var crypto = require("crypto");
+var config = require("config");
 
 module.exports = {
-  generateHashForPassword: function (email, password, sold) {
+  generateHashForPassword (email, password) {
     logger.data("[password.generateHashForPassword, ", email, "] Try generate hash.");
     var sha1sum = crypto.createHash("sha1");
-    sha1sum.update(password + email + sold);
+    sha1sum.update(password + email + config.app.secret);
```

Comment:
We start using global config varibale in utility. As a result we also updating config in tests... I think we should still have `sold` for `generateHashForPassword` and add sold arg for `isPasswordCorrect`